mpv.el

```diff
@@ -395,14 +395,9 @@ https://mpv.io/manual/master/#command-interface-playlist-play-index"
       (pcase pos
 	("current" (mpv-seek 0))
 	((and (pred numberp) (pred (= cur))) (mpv-seek 0))
-	("none" (mpv-set-property "pause" "no"))
+	("none" (mpv-set-property "pause" "yes"))
```

Comment:
In principle we should even enter idle mode here (according to the 0.33 man page), but I'm not sure how to do that on earlier versions. In 0.33 `(mpv-set-property "playlist-pos" -1)` would have that effect, but I doubt that this will work on earlier versions. So I think your current implementation is probably as close as we can get.